lib/modules/dosomething/dosomething_user/dosomething_user.module

```diff
@@ -757,6 +757,29 @@ function dosomething_user_is_under_thirteen($user = NULL) {
 }
 
 /**
+ * Checks that user will be 18 by a certain date
+ *
+ * @param $date
+ *
+ * @return bool
+ *  Returns true if user will be 18 by the given date, else false.
+ */
+function dosomething_user_will_be_eightteen_by_date($date, $user = NULL) {
```

Comment:
@angaither I couldn't find that function but I switched this one to be more general. Now it just returns the age of a user on the given date. Seems like that might be worth having but I'm down to remove